fastlane/Fastfile

```diff
@@ -669,12 +736,17 @@ REPOSITORY_NAME="WordPress-Android"
       sh("./gradlew clean >> #{logfile_path} 2>&1")
 
       sh("mkdir -p #{build_dir}")
-      UI.message("Running lint...")
-      sh("echo \"Running lint...\" >> #{logfile_path}")
-      sh("./gradlew lint#{options[:app]}#{options[:flavor]}Release >> #{logfile_path} 2>&1") unless is_ci
+      if options[:skip_lint].nil?
+        UI.message("Running lint...")
+        sh("echo \"Running lint...\" >> #{logfile_path}")
+        sh("./gradlew lint#{app}#{options[:flavor]}#{options[:buildType]} >> #{logfile_path} 2>&1") unless is_ci
```

Comment:
💄  nitpick that is really non-blocker and can **totally** only be addressed in separate future PR:

It would be nice to have a helper method in that Fastfile to assemble different strings as CamelCase to build the gradle task with the right case. This is not a big deal at all since apparently gradle is case-insensitive so this works too, it would just improve readability both of the code in Fastfile and in the logs 🙂 

There's a `capitalize` method on `String` in ruby, so you can use this to turn `jetpack` into `Jetpack` and `vanilla` into `Vanilla`. This could look like this for example:

```ruby
def gradle_task(*args)
  ([args.first.downcase] + args[1..].map(&:capitalize)).join
end
```
and called like this:
```ruby
sh("./gradlew #{gradle_task('lint', app, options[:flavor], options[:buildType])} >> #{logfile_path} 2>&1") unless is_ci
```

Not tested, and again, you can keep that as a separate improvement in different future PR.

Reply:
Also tracking this in https://github.com/wordpress-mobile/WordPress-Android/issues/14874